apps/core-app/src/Routes.tsx

```diff
@@ -30,7 +30,7 @@ const Routes = () => {
         <Route path="proposal/:proposalId" element={<Proposals />} />
         <Route path="vaults" element={<Vaults />} />
         <Route path="members" element={<Members />} />
-        <Route path="member/:memberAddress" element={<Members />} />
+        <Route path="members/:memberAddress" element={<Member />} />
```

Comment:
typo - this should be `member/:memberAddress`

Reply:
oh i thought in restful routing you used the plural.

Reply:
oh my bad - yeah you do, i think i saw members on the line above and thought it was duplicated here - ignore this